src/rules/package-lavamoat-tsup-conforms.ts

```diff
@@ -0,0 +1,48 @@
+import { buildRule } from './build-rule';
+import { RuleName } from './types';
+
+export default buildRule({
+  name: RuleName.PackageLavamoatTsupConform,
+  description:
+    'Does the `lavamoat.allowscripts` field in `package.json` conform?',
+  dependencies: [RuleName.RequireValidPackageManifest],
+  execute: async ({ project, template, pass, fail }) => {
+    const entryPath = 'package.json';
+    const templateManifest = await template.fs.readJsonFile(entryPath);
+    const projectManifest = await project.fs.readJsonFile(entryPath);
+
+    type Key = keyof typeof templateManifest;
+    const templateLavamoat = templateManifest?.['lavamoat' as Key];
```

Comment:
Ah, this is because `undefined` isn't a valid JSON value, and so the way this changes `PackageManifestSchema` makes it conflict with the `Json` type. `@metamask/utils` would usually give us a way around this, but it doesn't in this case, and so I'll have to update it to make this work. In the meantime this is fine!